src/test/java/nl/stil4m/mollie/concepts/PaymentsIntegrationTest.java

```diff
@@ -1,35 +1,33 @@
 package nl.stil4m.mollie.concepts;
 
-import nl.stil4m.mollie.Client;
-import nl.stil4m.mollie.ResponseOrError;
-import nl.stil4m.mollie.domain.CreatePayment;
-import nl.stil4m.mollie.domain.Issuer;
-import nl.stil4m.mollie.domain.Page;
-import nl.stil4m.mollie.domain.Payment;
-import nl.stil4m.mollie.domain.subpayments.ideal.CreateIdealPayment;
-import nl.stil4m.mollie.domain.subpayments.ideal.IdealPaymentOptions;
-import org.apache.http.HttpStatus;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.Ignore;
-
 import java.io.IOException;
 import java.net.URISyntaxException;
 import java.util.Date;
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Optional;
-
+import nl.stil4m.mollie.Client;
+import nl.stil4m.mollie.ResponseOrError;
 import static nl.stil4m.mollie.TestUtil.TEST_TIMEOUT;
 import static nl.stil4m.mollie.TestUtil.VALID_API_KEY;
 import static nl.stil4m.mollie.TestUtil.assertWithin;
 import static nl.stil4m.mollie.TestUtil.strictClientWithApiKey;
+import nl.stil4m.mollie.domain.CreatePayment;
+import nl.stil4m.mollie.domain.Issuer;
+import nl.stil4m.mollie.domain.Page;
+import nl.stil4m.mollie.domain.Payment;
+import nl.stil4m.mollie.domain.subpayments.ideal.CreateIdealPayment;
+import nl.stil4m.mollie.domain.subpayments.ideal.IdealPaymentOptions;
+import org.apache.http.HttpStatus;
```

Comment:
It seems that your import order settings are not set up the same. Can you layout static imports separately?

My setup is:
```
com.*

nl.*

org.*

javax.*

java.*

other imports

other static imports

```